redis-om-spring/src/main/java/com/redis/om/spring/metamodel/indexed/DateField.java

```diff
@@ -4,17 +4,27 @@
 
 import com.redis.om.spring.metamodel.MetamodelField;
 import com.redis.om.spring.search.stream.predicates.numeric.BetweenPredicate;
+import com.redis.om.spring.search.stream.predicates.numeric.EqualPredicate;
 import com.redis.om.spring.search.stream.predicates.numeric.GreaterThanOrEqualPredicate;
 import com.redis.om.spring.search.stream.predicates.numeric.GreaterThanPredicate;
 import com.redis.om.spring.search.stream.predicates.numeric.LessThanOrEqualPredicate;
 import com.redis.om.spring.search.stream.predicates.numeric.LessThanPredicate;
+import com.redis.om.spring.search.stream.predicates.numeric.NotEqualPredicate;
```

Comment:
In ULIDIdentifiereGenerator I see that we specifically ensure ULID, ... integer and long.

But here, we only index on DateField, GeoField, and TagField. Can we add Numeric? I recognize we don't (yet) have Vector available here.

Reply:
No, I haven't tackled Vector yet.